tests/platform_tests/test_memory_exhaustion.py

```diff
@@ -22,9 +22,17 @@ class TestMemoryExhaustion:
     """
     This test case is used to verify that DUT will reboot when it runs out of memory.
     """
+    def wait_lc_healthy_if_sup(duthost, duthosts):
```

Comment:
is this function in class. missing self?

Reply:
thanks for reminding! done